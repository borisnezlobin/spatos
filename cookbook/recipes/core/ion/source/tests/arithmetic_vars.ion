echo '# ANCHOR: individual_assignments'
let value = 5
echo $value
let value += 5
echo $value
let value -= 2
echo $value
let value *= 2
echo $value
let value //= 2
echo $value
let value **= 2
echo $value
let value /= 2
echo $value
echo '# ANCHOR_END: individual_assignments'
echo '# ANCHOR: multiple_assignments'
let a b = 5 5
echo $a $b
let a b += 5 5
echo $a $b
let a b -= 2 2
echo $a $b
let a b *= 2 2
echo $a $b
let a b //= 2 2
echo $a $b
let a b **= 2 2
echo $a $b
let a b /= 2 2
echo $a $b
echo '# ANCHOR_END: multiple_assignments'
