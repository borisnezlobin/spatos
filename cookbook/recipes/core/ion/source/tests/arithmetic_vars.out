# ANCHOR: individual_assignments
5
10
8
16
8
64.0
32.0
# ANCHOR_END: individual_assignments
# ANCHOR: multiple_assignments
5 5
10 10
8 8
16 16
8 8
64.0 64.0
32.0 32.0
# ANCHOR_END: multiple_assignments
